spyderlib/spyder.py

```diff
@@ -580,6 +580,14 @@ def setup(self):
                                             context=Qt.WidgetShortcut)
             self.register_shortcut(self.replace_action, "Editor",
                                    "Replace text")
+            self.file_switcher_action = create_action(self, _('File switcher...'),
+                                            icon='filelist.png',
+                                            tip=_('Fast switch between files'),
+                                            triggered=self.call_file_switcher,
+                                            context=Qt.ApplicationShortcut)
+            self.register_shortcut(self.file_switcher_action, "_",
+                                   "file switcher")

```

Comment:
`file switcher` -> `File switcher`. Don't worry, shortcut names are picked up correctly in `config.py`